src/Chronos.php

```diff
@@ -563,27 +563,21 @@ public static function create(
         ?int $year = null,
         ?int $month = null,
         ?int $day = null,
-        ?int $hour = null,
-        ?int $minute = null,
-        ?int $second = null,
-        ?int $microsecond = null,
+        ?int $hour = 0,
```

Comment:
So time parts now default to `0` but the date parts still default to `null` aka based on current date? Doesn't that seem inconsistent?

Reply:
The date seems like it should be required in both create() and createFromDate(), yes?

Reply:
Having the date parts default to 0 would be pretty unpleasant and a larger break in compatibility. 

> The date seems like it should be required in both create() and createFromDate()

Defaulting to 'now' is a good default value that we should continue to use.